build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     // set a valid scala suffix for the buildscript itself
-    project.ext.scalaSuffix = '_2.12'
+    project.ext.scalaSuffix = '_2.13'
```

Comment:
```suggestion
    project.ext.scalaSuffix = '_2.12'
```

Reply:
looks like the only place this is currently used is line 24, thought it might be easier to ensure any scala deps used by the buildscript itself would be easier to track this way